src/app/components/product/product.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="products">
+    <a class="products__container" [routerLink]="'/product/' + product.id">
+      <img
+        [src]="'http:' + product.api_featured_image"
+        width="210"
+        height="210"
+        class="image"
+        id="test-image"
+      />
+      <p class="brand_name">{{ product.brand }}</p>
+      <p>{{ product.name }}</p>
+      <p>
+        {{ product.price | currency }}
+      </p>
+      <!-- <Colors /> -->
+      <ul class="colors" *ngIf="product.product_colors">
+        <li *ngFor="let color of product.product_colors.slice(0, 30)"
+          class="color_circle"
+          [ngStyle]="{ 'background': color.hex_value }"
+          [id]="color.hex_value"
+        ></li>
+      </ul>
+    </a>
+    <!-- <Buttons /> -->
+    <div class="button">
+      <button
+        type="submit"
+        [ngClass]="isAddedToCart ? 'button_buy__active' : 'button_buy'"
```

Comment:
Minor:
For code purity, it is better to define such things in the ts file, and pass the value here, for example, using a getter/
`get submitBtmClass(): string {
   return this.isAddedToCart ? 'button_buy__active' : 'button_buy'
}
[ngClass]="submitBtmClass"
`